docs/userGuide/deployingTheSite.md

```diff
@@ -52,18 +53,45 @@ You can override the default deployment settings %%(e.g., repo/branch to deploy)
 ### Deploying to GitHub Pages via Travis CI
 **You can setup [<tooltip content="a platform for Continuous Integration and Delivery (among other things)">Travis CI</tooltip>](https://www.travis-ci.org/) to automatically build and deploy your site on GitHub Pages every time your GitHub repo is updated.**
 
-Here are the steps to set up Travis CI:
 
-1. [Sign up with GitHub](https://travis-ci.com/signin) at [Travis CI](https://travis-ci.com).
+#### Adding your repository to Travis CI
+
+<panel header="{{icon_info}} Travis CI Migration" type="info" expanded>
+
+As of May 2018, Travis CI was [undergoing migration to `travis-ci.com`](https://docs.travis-ci.com/user/migrate/open-source-on-travis-ci-com/), which changes the way you setup repositories on Travis CI. 
+
+- If you are new to Travis CI, you should **add your repository on `travis-ci.com`**.
+- Alternatively, if you are already using Travis CI on `travis-ci.org`, you can continue to **add your repository on `travis-ci.org`**.
+</panel>
+<p/>
+<tabs>
+  <tab header="Add your repository on `travis-ci.com`">
+
+1. [Sign in to Travis](https://travis-ci.com/signin) using your GitHub account.
+1. Accept the authorisation for Travis CI when you are redirected to GitHub.
+1. Go to the [Repositories page](https://travis-ci.org/account/repositories), and click on the green _Activate_ button.
```

Comment:
`https://travis-ci.com/account/repositories` (not `.org`)

Reply:
Er also I don't see the green _Activate_ button on my screen. :X

Reply:
> Er also I don't see the green _Activate_ button on my screen. :X

Hmm, it shows up for me 🤔:

![Green Activate button](https://user-images.githubusercontent.com/1782590/52932689-8d8fe800-338b-11e9-80de-a001444cd13f.png)

It will only show on `travis-ci.com` though (https://travis-ci.com/account/repositories).

Reply:
URL corrected.

Reply:
> Hmm, it shows up for me 🤔:

Ah OK, I think it is because for my case, I have used travis-ci.com before and already migrated over to GitHub Apps Integration, so I don't have to press the "Activate" button. Seems like active users should be pressing "Manage repositories on GitHub" instead to modify the repository list.

![image](https://user-images.githubusercontent.com/3168908/52997151-aa472100-345a-11e9-973b-8255db32831c.png)

Reply:
@yamgent good point, added a clarifying instruction.